_config.yml

```diff
@@ -98,28 +101,22 @@ custom:
 
 
 # Build settings
-safe: false
 source: src
 destination: publish
-incremental: true
 bundler_args: --without production
-markdown: kramdown
-excerpt_separator: "~~~"
 
-include: ["_pages"]
 exclude: ["diagrams", "appengine", "tests", "site"]
 
 plugins:
 - jekyll-assets
 - jekyll-toc
 - jekyll-sitemap
 
-
 assets:
-  compress:
-    css: true # default - development: false, production: true
-    js: true # default - development: false, production: true
-  cache: .asset-cache # default: .asset-cache
+  # compress:
```

Comment:
Why comment these out?

Reply:
It is easier to "debug" locally when using the defaults.

Production settings aren't affected (oops, they currently are: I just need to add `JEKYLL_ENV=production` to the Travis settings. I'll do that now).

Reply:
Done